scripts/generate/mockgen.sh

```diff
@@ -43,4 +44,4 @@ EOF
 
 mkdir -p $(dirname ${outputfile})
 
-echo "$data" | goimports > "${outputfile}"
+echo "$data" | sed -e "s/${PROJECT_VENDOR}//" | goimports > "${outputfile}"
```

Comment:
You can simplify this by changing the `sed` delimiter to something other than `/`.

```bash
PROJECT_VENDOR="github.com/awslabs/amazon-ecr-credential-helper/ecr-login/vendor/"
# ...
echo "$data" | sed -e "s|${PROJECT_VENDOR}||" | goimports > "${outputfile}"
```